### DISCUSSION TITLE
Detection percentage as an entity attribute in Home Assistant

### DISCUSSION THREAD

**Discussion by uscan27** [blakeblackshear/frigate#11881] [Ideas]
Hello,

I'm trying to protect my home using frigate to trigger an alarm if a human is detected in my garden when I'm not home. I try to reduce false positives, but still, a small amount remains.

I would like to able Home Assistant to decide wether ring the alarm or not depending on the percentage of human detection.
If detection is below 80% just notify on the smartphone, if above ring the alarm.

This information could be an attribute of the person occupancy entity . For exemple `binary_sensor.entree_nord_person_occupancy` could have a `detection` attribute between 0 and 100.

This small information would be greatly useful !

Thank you !

**Comment by NickM-27** [comment]
The occupancy sensor can be true for multiple objects. If you want this level of granularity you should use /events for notifications and not rely on the sensor

**Reply by uscan27** [reply]
But a `person_occupancy` entity reacts to human only.

/events is something I don't know....

**Reply by NickM-27** [reply]
right, but there can be 2 people detected at the same time, so it would either have to have an attribute that showed all of the scores or something else. Either way, more advanced use cases like this should use mqtt instead of basic sensors

**Reply by uscan27** [reply]
Those infos can be found in the mqtt payload? If yes I will dig into it... But this if a bit more that what i know for now, si I would have to learn more... That's why an attribute would have been easier for me (even if it gives an average score when multiple person are detected) !
Thank you for the idea of looking into mqtt...

**Reply by NickM-27** [reply]
You can always just use the blueprint https://github.com/SgtBatten/HA_blueprints/

**Reply by uscan27** [reply]
I use it for notifications. But it cannot notify if person detection score is below 80, for example, and trigger an alarm if above?

**Reply by NickM-27** [reply]
sure it can, that can just be added as a filter

**Reply by uscan27** [reply]
How do you do that? I just had a look into the blueprint I don't see how ?

**Reply by TheGlorySaint** [reply]
@uscan27 Do you found a Solution? I'm currently in your State where i want to get the percentage.
